src/full/Agda/Interaction/BasicOps.hs

```diff
@@ -88,9 +88,11 @@ import Agda.Utils.Pretty as P
 import Agda.Utils.Permutation
 import Agda.Utils.Size
 import Agda.Utils.String
-
+import Agda.Utils.Tuple
 import Agda.Utils.Impossible
 
+import Data.List.NonEmpty (toList)
```

Comment:
should I expose it via Agda.Utils.List1 ?

Reply:
Would me more uniform...